beginning.md

```diff
@@ -11,28 +11,28 @@ Python 3.7.3 (default, Mar 27 2019, 13:36:35)
 Type "help", "copyright", "credits" or "license" for more information.
 >>> 
 ```
- * `>>>` known as python prompt.
- * Use `CTRL + D` to exit from the terminal or `exit()` command.
+ * `>>>` is known as python prompt.
+ * Use `CTRL + d` to exit from the terminal or `exit()` command.
  * After quitting, all programs will get destroyed.
  
 ## My first program
-* `print()` method is used to display output like `printf` in `C language`.
+* `print()` method is used to display output like `printf` in *C language*.
 
 ```
 >>> print('Welcome to FOSSMEET, PUNE')
 Welcome to FOSSMEET, PUNE
 ```
 
-* We can also print in multiple lines by using *\n*.
+* We can also print in multiple lines by using `\n`.
 ```
 >>> print('Welcome to FOSSMEET, PUNE\n Let\'s Learn Python 3')
 Welcome to FOSSMEET, PUNE
  Let's Learn Python 3
```

Comment:
Here \n is not used.

Reply:
Sorry I meant to say we need to edit next line to improve uses of escape.

Reply:
I dont understand what you mean.

Reply:
aaaaaaaaahhh I understand what you mean...
```
In [1]: print('Welcome to FOSSMEET, PUNE\n Let\'s Learn Python 3')                                                                                
Welcome to FOSSMEET, PUNE
 Let's Learn Python 3

In [2]: print("Welcome to FOSSMEET, PUNE\n Let's Learn Python 3")                                                                                 
Welcome to FOSSMEET, PUNE
 Let's Learn Python 3

```

`\s` right? No need of this when use use default as `"string's"`.
I don't think you need to correct it.